src/test/java/org/plumelib/util/MathPlumeTest.java

```diff
@@ -62,7 +62,9 @@ private static int[] intIteratorArray(Iterator<Integer> itor) {
     }
     int[] a = new int[v.size()];
     for (int i = 0; i < a.length; i++) {
-      a[i] = v.get(i).intValue();
+      @SuppressWarnings("index:argument")
```

Comment:
Can you eliminate this warning suppression by declaring `a` and `v` to be the same length?